build-ps/jenkins/build-binary

```diff
@@ -0,0 +1,164 @@
+else
+    CMAKE_OPT+=" -DMYSQL_MAINTAINER_MODE=OFF"
+fi
+if [[ ${CMAKE_OPTS} == *WITH_VALGRIND=ON* ]]; then
+    BUILD_COMMENT+="-valgrind"
+fi
+
+
+# ------------------------------------------------------------------------------
+# set version
+# ------------------------------------------------------------------------------
+source "$SOURCEDIR/VERSION"
+if [[ -n $(which git) ]] && [[ -d "$SOURCEDIR/.git" ]]; then
+    REVISION="$(cd "$SOURCEDIR"; git rev-parse --short HEAD)"
+fi
+MYSQL_VERSION="$MYSQL_VERSION_MAJOR.$MYSQL_VERSION_MINOR.$MYSQL_VERSION_PATCH"
+PERCONA_SERVER_VERSION="$(echo $MYSQL_VERSION_EXTRA | sed 's/^-//')"
+PRODUCT="Percona-Server-$MYSQL_VERSION-$PERCONA_SERVER_VERSION"
+TOKUDB_BACKUP_VERSION="${MYSQL_VERSION}${MYSQL_VERSION_EXTRA}"
+PRODUCT_FULL="Percona-Server-${MYSQL_VERSION}-${PERCONA_SERVER_VERSION}${BUILD_COMMENT}-${TAG}$(uname -s)${DIST_NAME}.${TARGET_ARCH}${SSL_VER}"
+COMMENT="Percona Server (GPL), Release ${MYSQL_VERSION_EXTRA#-}, Revision ${REVISION:-}${BUILD_COMMENT}"
+
+
+# ------------------------------------------------------------------------------
+# Check tokudb
+# ------------------------------------------------------------------------------
+if [[ ${WITH_TOKUDB} == ON ]]; then
+    CMAKE_OPTS+=" -DBUILD_TESTING=OFF -DUSE_GTAGS=OFF -DUSE_CTAGS=OFF -DUSE_ETAGS=OFF -DUSE_CSCOPE=OFF -DTOKUDB_BACKUP_PLUGIN_VERSION=${TOKUDB_BACKUP_VERSION}"
+    if [[ ${CMAKE_BUILD_TYPE} == Debug ]]; then
+        CMAKE_OPTS+=" -DTOKU_DEBUG_PARANOID=ON"
+    else
+        CMAKE_OPTS+=" -DTOKU_DEBUG_PARANOID=OFF"
+    fi
+    if [[ $CMAKE_OPTS == *WITH_VALGRIND=ON* ]]; then
+        CMAKE_OPTS+=" -DUSE_VALGRIND=ON"
+    fi
+fi
+
+
+# ------------------------------------------------------------------------------
+# Check mecab
+# ------------------------------------------------------------------------------
+if [ -f /etc/mecabrc ]; then
+    WITH_MECAB="system"
+else
+    WITH_MECAB=""
+fi
+
+
+# ------------------------------------------------------------------------------
+# Set compile options
+# ------------------------------------------------------------------------------
+if [[ $COMPILER != default ]]; then
+    export CC=${COMPILER}
+    export CXX=${COMPILER/cc/++}
+else
+    export CC=gcc
+    export CXX=g++
+fi
+export LSAN_OPTIONS=verbosity=2:log_threads=1
+
+
+# ------------------------------------------------------------------------------
+# Finaly, compile!
+# ------------------------------------------------------------------------------
+pushd ${WORKDIR}
+    ${JOB_CMAKE} \
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
+        ${CMAKE_OPT} \
+        ${ANALYZER_OPT} \
+        -DWITH_TOKUDB=${WITH_TOKUDB} \
+        -DWITH_ROCKSDB=${WITH_ROCKSDB} \
+        -DWITH_EMBEDDED_SERVER=${WITH_EMBEDDED_SERVER} \
+        -DWITH_RAPID=${WITH_RAPID} \
+        -DWITH_SSL=${WITH_SSL} \
+        -DWITH_MECAB=${WITH_MECAB} \
+        -DWITHOUT_PERFSCHEMA_STORAGE_ENGINE=${WITHOUT_PERFSCHEMA_STORAGE_ENGINE} \
```

Comment:
The same comment re `WITHOUT_PERFSCHEMA_STORAGE_ENGINE`

Reply:
I created PERFSCHEMA_OPTS variable which passed to cmake and can contain -DWITHOUT_PERFSCHEMA_STORAGE_ENGINE=ON value or empty value (default)